### PR TITLE
scenario-04 - Rabbitmq outage

### PR DESCRIPTION
**Pull request by saurabh3460**

### CODE REVIEW

File: scenarios/scenario-04/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Scenario: 
+
```
Comment (Hashfyre) [inline scenarios/scenario-04/README.md:2]: Flesh out the scenario description and operating paradigms / constraints.

**Review by Hashfyre — CHANGES_REQUESTED** [review]

File: app/robot-shop/helm/values.yaml [hunk 2]
```diff
@@ -127,6 +127,13 @@ payment:
...
 rabbitmq:
   replicas: 3
+  resources:
+    requests:
+      cpu: 500m
+      memory: 2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
   tolerations:
```
Comment (Hashfyre) [inline app/robot-shop/helm/values.yaml:137]: Why are we doing workload specific tolerations? RabbitMQ should tolerare the `key: persistent, value: true` taint on the `persistent-ng` nodegroup.

**Review by Hashfyre — CHANGES_REQUESTED** [review]

File: scenarios/scenario-04/README.md (hunk 1 above)
Comment (saurabh3460) [reply scenarios/scenario-04/README.md:2]: will create a separate PR for this, where will rename it to persistence and move all app persistence layer to it

File: app/robot-shop/helm/values.yaml (hunk 2 above)
Comment (saurabh3460) [reply app/robot-shop/helm/values.yaml:137]: will create a separate PR for this, where will rename it to persistence and move all app persistence layer to it

**Comment by Hashfyre** [comment]
Way to do capacity planning:
- First apply resources and limits to rabbitmq-loadgen (the least you can, the container/pod just runs a CLI command)
- let it be OOMKilled, then add a bit more CPU and/or memory
- total mem/cpu required: `400 x {resources{}} / 3 (max nodes) = closest CPU/Memory` configuration of the instance we need for `loadgen-ng`
- we also add resources required for the embedded `load` deployment for robot-shop (1 pod, with some resources)

**Review by Hashfyre — APPROVED** [review]
LGTM

**Pull request merged**